src/Illuminate/Queue/Worker.php

```diff
@@ -723,10 +760,19 @@ protected function maxAttemptsExceededException($job)
      * Sleep the script for a given number of seconds.
      *
      * @param  int|float  $seconds
+     * @param  string $connectionName
+     * @param  int $sleepingType
      * @return void
      */
-    public function sleep($seconds)
+    public function sleep($seconds, $connectionName, $sleepingType)
```

Comment:
This is a breaking change to the method signature. Please undo this or send the PR to master.

Reply:
Can we fix this by adding a `sleepWithEvents()` method, or is that going to litter the code base?